SW4STM32/BellaLui/Application/HostBoard/Src/Sensors/UnbiasedSensor.cpp

```diff
@@ -65,7 +65,73 @@ bool UnbiasedSensor<T>::fetch(T* data) {
 
 template<class T>
 uint8_t UnbiasedSensor<T>::removeOutsiders(float** data) {
-	// TODO
+	uint8_t sorting_array[SENSORS_NB] = {0,1,2,3}; //TODO:magic numbers? could work with array of pointers of data instead of indexes... faster code I believe but more RAM (insignificant?)
```

Comment:
You could initialize the array with the std::iota function instead of hardcoding the values: (	https://en.cppreference.com/w/cpp/algorithm/iota)
```suggestion
std::iota(sorting_array, sorting_array + SENSORS_NB, 0);
```